meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -173,61 +203,90 @@ if [ -z $slug ]; then
 fi
 log "Found slug $slug for this device."
 
+# Run hacks
+runhacks
+
+# Detect containers engine
+if which docker &>/dev/null; then
+    DOCKER=docker
+else if which rce &>/dev/null; then
+    DOCKER=rce
+else
+    log ERROR "Can't detect the containers engine on the host OS."
+fi
+
+# Supervisor update
+if [ ! -z "$UPDATER_SUPERVISOR_TAG" ]; then
+    log "Supervisor update requested through arguments ."
+    resin-device-progress --percentage 25 --state "Host OS Update: Done. Updating supervisor..."
```

Comment:
This progress value seems not be based on any actual progress. Can progress not be accurately detected for the update?